tests/src/Core/Lock/CacheLockDriverTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Friendica\Test\src\Core\Lock;
```

Comment:
Can we avoid the `src` namespace? Is there a reason why we would need this intermediate folder in the `tests` directory?